plugins/modules/openssh_keypair.py

```diff
@@ -60,11 +60,12 @@
         description:
             - Passphrase used to decrypt an existing private key or encrypt a newly generated private key.
             - Passphrases are not supported for I(type=rsa1).
+            - Can only be used when C(backend=cryptography) or C(backend=auto) and the OpenSSH binary is not installed
```

Comment:
```suggestion
            - Can only be used when I(backend=cryptography), or when I(backend=auto) and a required C(cryptography) version is installed. 
```

Reply:
No matter how many times I check I still miss these, thanks. Will add a changelog fragment and open for review after I take a final look at this. And once this is ready for review I'll take some time to look at #150.